Apps/Src/ReadTritium.c

```diff
@@ -17,8 +17,11 @@
 
 
 uint16_t Motor_FaultBitmap = T_NONE;
-static float Motor_RPM = CAR_STOPPED; //Car is stopped until velocity is read
-static float Motor_Velocity = CAR_STOPPED; //^^^^
+static float Motor_RPM = CAR_STOPPED;		//Car is stopped until velocity is read
+static float Motor_Velocity = CAR_STOPPED;	//^^^^
```

Comment:
don't do the ^^^^ comment. You can just adjust the first one to say "until velocity and RPM are read"